src/helpers/shared/pagination.js

```diff
@@ -1,16 +1,11 @@
 export const defaultSettings = {
-  limit: 50,
-  offset: 0,
-  itemCount: 1,
```

Comment:
I think that the default settings should have `itemCount` at least set to 0 so if the server reposnds with something strange we won't break pagination.